app.js

```diff
@@ -1,93 +1,109 @@
-if (focus.toLowerCase() === 'yes') {
+function focus(){
+  var focus = prompt('Is Python the computer language you think I\'m focusing my studies?');
+  if (focus.toLowerCase() === 'yes') {
     alert('Impressive, you are on the money!');
     console.log(focus);
-} else {
+  } else {
     alert('Actually, I\'m focusing my study on Python, as it is apparently the current standard in the GIS industry');
     console.log(focus);
-}
+  }
+} focus();
 
-var failedAttempts = [];
-var computerChoice = Math.floor((Math.random() * 10) + 1);
-var guessNumber = prompt('Guess a number between 1 and 10.');
-for (var i = 1; i < 4; i++) {
-  if (guessNumber === null) {
+//
+function functionName() {
+  var failedAttempts = [];
+  var computerChoice = Math.floor((Math.random() * 10) + 1);
+  var guessNumber = prompt('Guess a number between 1 and 10.');
+  for (var i = 1; i < 4; i++) {
+    if (guessNumber === null) {
       alert('You did not guess a number.');
       console.log(guessNumber);
       var guessNumber = prompt('Please guess a number.');
       alert(failedAttempts + ' was guessed already');
       continue;
-} if (guessNumber === computerChoice) {
+    } if (guessNumber === computerChoice) {
       alert('You guessed it!');
       console.log(guessNumber);
       alert(failedAttempts + ' was guessed already');
       break;
-} else if (guessNumber < computerChoice) {
+    } else if (guessNumber < computerChoice) {
       alert('Try Again!');
       failedAttempts.push(guessNumber);
       alert(failedAttempts + ' was guessed already');
       var guessNumber = prompt('Please guess another number between ' + guessNumber + ' and 10');
       continue;
-} else if (guessNumber > computerChoice) {
+    } else if (guessNumber > computerChoice) {
       alert('Try again! Pick another number between 1 and ' + guessNumber);
       failedAttempts.push(guessNumber);
       alert(failedAttempts + ' was guessed already');
       var guessNumber = prompt('Please pick another number between 1 and ' + guessNumber);
       continue;
     }
   }
+} functionName();
 
-var userAnswers = [];
-var userGuess = prompt('Can you guess the state where I was born?');
-    userAnswers.push(userGuess);
-for (var i = 1; i < 6; i++) {
-  console.log(i);
-  if (userGuess.toLowerCase() === 'california') {
+function userGuess() {
+  var userAnswers = [];
+  var userGuess = prompt('Can you guess the state where I was born?');
+  userAnswers.push(userGuess);
+  for (var i = 1; i < 6; i++) {
+    console.log(i);
+    if (userGuess.toLowerCase() === 'california') {
       alert('You are good! I was in fact born in California.');
       console.log(userGuess);
       break;
-  } else if (userGuess.toLowerCase() !== 'california') {
+    } else if (userGuess.toLowerCase() !== 'california') {
       alert('Sorry, try again!');
       console.log(userGuess);
       var userGuess = prompt('Guess another state please.');
       continue;
+    }
   }
-}
+} userGuess();
```

Comment:
your function declarations should end with a semicolon, and your function call should be on its own line